package.json

```
@@ -19,7 +19,7 @@
     "check": "npm-run-all prettier:fix lint:fix test build",
     "dev:check": "npm-run-all check dev",
     "predev": "bash -c 'f() { if [ $(command -v podman) ]; then podman compose up -d valkey; else docker compose up -d valkey; fi }; f'",
-    "dev": "nodemon --watch src -e ts,ejs,json,scss --exec npm run start",
+    "dev": "ts-node-dev --respawn --transpile-only src/server.ts | pino-colada",
     "prelocalstack": "bash -c 'f() { if [ $(command -v podman) ]; then podman compose up -d valkey db blobstorage backend; else docker compose up -d valkey db blobstorage backend; fi }; f'",
     "localstack": "trap 'exit 0' SIGINT; nodemon --watch src -e ts,ejs,json,scss --exec npm run start",
```

We'll need to update the localstack script as well

```suggestion
    "localstack": "trap 'exit 0' SIGINT; ts-node-dev --respawn --transpile-only src/server.ts | pino-colada",
```